### ISSUE TITLE
Adding animations to the Page on loading and scrolling

### ISSUE THREAD

**Issue by MHusain22** [akshitagupta15june/Moksh#1561]
I can Add animations to the Page on loading and scrolling just like in my portfolio website https://mhportfolio.vercel.app/. Please assign me this to add new feature.

- I have checked the existing issues *
- I have read the Contributing Guidelines *
- I have starred the repository *
- Working under Gssoc'24

**Comment by akshitagupta15june** [comment]
can u suggest some animations that you are planning to implement, you can share in discord channel, link is mentioned in readme.md

**Comment by MHusain22** [comment]
yes, I can add animations in vanilla CSS and a library of animations data-aos https://michalsnik.github.io/aos/

**Comment by Skb08** [comment]
@akshitagupta15june  If this task is not completed, can you please assign it to me?
